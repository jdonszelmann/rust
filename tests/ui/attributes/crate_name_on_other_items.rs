//@compile-flags: --crate-type=lib
// reproduces #137687
// FIXME(jdonszelmann): should ERROR malformed `crate_name` attribute input but now still ignored.
// This is for the beta backport of 1.87
#[crate_name = concat!("Cloneb")]

macro_rules! inline {
    () => {};
}

#[crate_name] //~ ERROR malformed `crate_name` attribute input
mod foo {}
