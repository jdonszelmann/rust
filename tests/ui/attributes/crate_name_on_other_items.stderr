error: malformed `crate_name` attribute input
  --> $DIR/crate_name_on_other_items.rs:11:1
   |
LL | #[crate_name]
   | ^^^^^^^^^^^^^ help: must be of the form: `#[crate_name = "name"]`

error: aborting due to 1 previous error

